immer/detail/arrays/no_capacity.hpp

```diff
@@ -159,10 +159,13 @@ struct no_capacity
     no_capacity push_back(T value) const
     {
         auto p = node_t::copy_n(size + 1, ptr, size);
-        IMMER_TRY {
+        IMMER_TRY
+        {
```

Comment:
The new line before the brace looks weird. Do you think there is any way to convince `clang-format` to move the brace back into the previous line?